PyInstaller/building/api.py

```diff
@@ -424,7 +424,8 @@ def __init__(self, *args, **kwargs):
                                  "", "OPTION"))
 
             if self.versrsrc:
-                if not os.path.isabs(self.versrsrc):
+                if (not isinstance(self.versrsrc, versioninfo.VSVersionInfo)
+                        and not os.path.isabs(self.versrsrc)):
```

Comment:
This is hard to read, since the second line is indented more. Also to make the change easier to understand I suggest keeping `not os.path.isabs`…)` in the forst line, like this:
```
                if (not os.path.isabs(self.versrsrc) and
                    not isinstance(self.versrsrc, versioninfo.VSVersionInfo)):
```
What do you think?

Reply:
I think the indented line is easier to read, but that's up to you. Should I change that?

Unfortunately it is not possible to swap the two lines, because the `os.path.isabs()` method throws the error. For this reason it is necessary to test if it is a VSVersionInfo before checking the path.

Reply:
Re. swapping: IC.

Anyway, please change the indention to match our coding stye. Please also see  https://github.com/pyinstaller/pyinstaller/pull/4539#issuecomment-554780238. Thanks

Reply:
Done myself to get this finished.